src/GameLogic/PlugIns/CreateItemChatCommandPlugIn.cs

```diff
@@ -0,0 +1,146 @@
+            public byte Opt { get; set; }
+        }
+
+        /// <inheritdoc />
+        public string Key => CommandKey;
+
+        /// <summary>
+        /// Min character status to use command
+        /// </summary>
+        public int MinStatusRequirement => (byte)CharacterStatus.GameMaster;
+
+        /// <inheritdoc />
+        public void HandleCommand(Player player, string command)
+        {
+            var dropCoordinates = player.CurrentMap.Terrain.GetRandomDropCoordinate(player.Position, 1);
+            try
+            {
+                var arguments = MarshallArguments(command.Split(' ').Skip(1).ToList());
+                var item = CreateItem(player, arguments);
+                var droppedItem = new DroppedItem(item, dropCoordinates, player.CurrentMap, player);
+                player.CurrentMap.Add(droppedItem);
+                player.ViewPlugIns.GetPlugIn<IShowMessagePlugIn>()?.ShowMessage($"[GM][/item] {item} created", MessageType.BlueNormal);
+            }
+            catch (ArgumentException e)
+            {
+                player.ViewPlugIns.GetPlugIn<IShowMessagePlugIn>()?.ShowMessage(e.Message, MessageType.BlueNormal);
+            }
+        }
+
+        private static Item CreateItem(Player player, Arguments arguments)
+        {
+            var item = new TemporaryItem();
+            var itemDefinition = player.GameContext.Configuration.Items.FirstOrDefault(def => def.Group == arguments.Group && def.Number == arguments.Number);
+            if (itemDefinition == null)
+            {
+                throw new ArgumentException($"[GM][/item] {arguments.Group} {arguments.Number} does not exists");
+            }
+            item.Definition = itemDefinition;
+            item.Level = arguments.Level;
+            item.Durability = itemDefinition.Durability;
+            item.HasSkill = arguments.Skill;
+            
+            if (arguments.Opt > 0)
+            {
+                var optionLink = new ItemOptionLink
+                {
+                    ItemOption = item.Definition.PossibleItemOptions.SelectMany(o => o.PossibleOptions)
+                        .First(o => o.OptionType == ItemOptionTypes.Option),
+                    Level = arguments.Opt
+                };
+                item.ItemOptions.Add(optionLink);
+            }
+
+            if (arguments.Luck)
+            {
+                var optionLink = new ItemOptionLink
+                {
+                    ItemOption = item.Definition.PossibleItemOptions.SelectMany(o => o.PossibleOptions)
+                        .First(o => o.OptionType == ItemOptionTypes.Luck)
+                };
+                item.ItemOptions.Add(optionLink);
+            }
+
+            if (arguments.Exc > 0)
+            {
+                var excellentOptions = item.Definition.PossibleItemOptions.SelectMany(o => o.PossibleOptions)
+                    .Where(o => o.OptionType == ItemOptionTypes.Excellent)
+                    .Take(arguments.Exc);
```

Comment:
I think ```Take``` here isn't correct. The Parameter is probably meant as a bit-mask, like a flags enum?
If this is the case you'd need to do smth like this:
```
.Where(o => (o.Number & arguments.Exc) > 0)
```